src/actions/NotificationActions.ts

```diff
@@ -167,7 +173,7 @@ export function setDeviceSettings(data: DeviceUpdatePayload): ThunkAction<Promis
   return async (dispatch, getState) => {
     const state = getState()
 
-    const deviceToken = await messaging().getToken()
+    const deviceToken = 'foo' // await messaging().getToken()
```

Comment:
Debugging code?

Idea: We make this field optional on the server for most request, and if it's not provided, we don't update the row. That way we don't need to have it working simply to update the user's preferences.

Reply:
Oops, yes debugging code somehow snuck in there. I'll fix this, and as for the idea to make it optional, that sounds fine, but it would need to be scoped in a separate task.